packages/vats/src/core/startWalletFactory.js

```diff
@@ -238,8 +236,8 @@ export const startWalletFactory = async (
   });
 
   await Promise.all([
-    E(bridgeManager).register(walletBridgeId, handler),
-    E(E.get(econCharterStartResult).creatorFacet).addInstance(
+    E(bridgeManager).register(BRIDGE_ID.PROVISION, handler),
```

Comment:
> How do I make it happy ...?

undo this change too; put it back to `walletBridgeId`.

Reply:
done.